app/api.py

```diff
@@ -1,10 +1,21 @@
+from app import db
 from flask import jsonify
+from flask import request
+from .models import Payment
 from flask.views import MethodView
 
 
 class PaymentResource(MethodView):
-    def get(self):
-        return jsonify(ok=True)
-
     def post(self):
-        pass
+        content = request.json
+        if 'note' not in content:
+            raise KeyError
+        highest_id = db.session.query(db.func.max(Payment.id)).scalar()
+        if highest_id is None:
+            highest_id = 0
+        leading_zeros = ('0' * (10 - len(str(highest_id + 1))))
+        new_variable_symbol = leading_zeros + str(highest_id + 1)
+        p = Payment(variable_symbol=new_variable_symbol, note=content['note'])
+        db.session.add(p)
+        db.session.commit()
```

Comment:
As we talked via signal, variable symbol can be basically the id. So we can remove the `id` field from models and make `variable_symbol` primary key. 

By doing so, the code could be simplified to:

```py
p = Payment(note=content['note'])
db.session.add(p)
db.session.commit()
return jsonify(variable_symbol=p.variable_symbol)
```

As for the formatting, is it a requirement to pad the variable symbol to 10 digits? If I were the user, I'd much rather write 47 as variable symbol then 0000000047 and make a silly off by one error :smiley: 

If it is, here is a simpler version of the padding: `f'{47:010}'`

Reply:
> As for the formatting, is it a requirement to pad the variable symbol to 10 digits? If I were the user, I'd much rather write 47 as variable symbol then 0000000047 and make a silly off by one error 😃

Yeah, I wasn't sure if it has to be 10 digit length or not, but I found out that it doesn't have to be, will remove it asap